scripts/ingests/utils.py

```diff
@@ -65,6 +65,70 @@ def load_simpledb(db_file, RECREATE_DB=True):
     return db
 
 
+def find_in_simbad(sources, desig_prefix, source_id_index = None, verbose = False):
+    """
+    Function to extract source designations from SIMBAD
+
+    Parameters
+    ----------
+    sources
+        Sources names to search for in SIMBAD
+    desig_prefix
+        prefix to search for in list of identifiers
+    source_id_index
```

Comment:
What is source_id_index?

Reply:
The index at which the suffix starts. Eg, if I’m trying to the the numbers after “Gaia DR2” or after “2MASS J”.

Reply:
I’ll try to explain better in the doc string